tests/test_makefile.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -e
+
+TMPDIR="$(realpath "$(mktemp -d)")"
+SRCDIR="$(dirname "$0")/.."
+
+function cleanup() {
+    rm -rf "$TMPDIR"
+}
+trap cleanup EXIT
+
+cd "$SRCDIR"
+
+################################################################################
+# Normal install.
+################################################################################
+
+make -s DESTDIR="$TMPDIR/install" DEFINES_FILE="$SRCDIR"/mender_grubenv_defines.example install
+
+find "$TMPDIR/install" -type f -o -type l | sed -e "s,^$TMPDIR/install/,," | sort > "$TMPDIR/actual.log"
+cat > "$TMPDIR/expected.log" <<EOF
+boot/efi/EFI/BOOT/grub.cfg
+boot/efi/EFI/BOOT/mender_grubenv1/env
+boot/efi/EFI/BOOT/mender_grubenv1/lock
+boot/efi/EFI/BOOT/mender_grubenv1/lock.sha256sum
+boot/efi/EFI/BOOT/mender_grubenv2/env
+boot/efi/EFI/BOOT/mender_grubenv2/lock
+boot/efi/EFI/BOOT/mender_grubenv2/lock.sha256sum
```

Comment:
By inspecting the Makefile, I don't understand why we don't expect these in the `install-legacy-tools` test case.

Reply:
The first test is installing everything, whereas the second is only installing the tools. I suppose I could make an additional case which only installs the regular tools, to make this a bit clearer.

Reply:
Ah, of course. Got it.

I don't think we need to add an extra test for that.